diff --git a/requirements/edx/testing.txt b/requirements/edx/testing.txt
--- a/requirements/edx/testing.txt
+++ b/requirements/edx/testing.txt
@@ -203,7 +203,7 @@
 mock==1.0.1
 mongoengine==0.10.0
 more-itertools==4.2.0     # via pytest
-moto==0.3.1
+# moto==0.3.1 version not available anymore but we don't need to run the tests anyway
 mysql-python==1.2.5
 needle==0.5.0             # via bok-choy
 networkx==1.7

diff --git a/requirements/edx/testing.in b/requirements/edx/testing.in
--- a/requirements/edx/testing.in
+++ b/requirements/edx/testing.in
@@ -26,7 +26,7 @@
 freezegun                 # Allows tests to mock the output of assorted datetime module functions
 httpretty                 # Library for mocking HTTP requests, used in many tests
 isort                     # For checking and fixing the order of imports
-moto==0.3.1               # Lets tests mock AWS access via the boto library
+# moto==0.3.1             # Version not available anymore but we don't need to run the tests anyway
 nose                      # Former test runner, we're still using some utility functions from it
 pa11ycrawler              # Python crawler (using Scrapy) that uses Pa11y to check accessibility of pages as it crawls
 pycodestyle               # Checker for compliance with the Python style guide (PEP 8)

diff --git a/requirements/edx/development.txt b/requirements/edx/development.txt
--- a/requirements/edx/development.txt
+++ b/requirements/edx/development.txt
@@ -212,7 +212,7 @@
 modernize==0.6.1
 mongoengine==0.10.0
 more-itertools==4.2.0
-moto==0.3.1
+# moto==0.3.1 version not available anymore but we don't need to run the tests anyway
 mysql-python==1.2.5
 needle==0.5.0
 networkx==1.7
